services/routes-google.js

```diff
@@ -84,17 +124,17 @@ const requestAll = async (origin, destination, vehicleName) => {
   const sections = findSection(dataGoogle.steps, missingTolls);
   const tolls = [];
 
-  for (const section in sections) {
+  // console.log(sections.tolls)
+  for (const section in sections.tolls) {
     isCache = false;
-    const startSection = sections[section].start_location;
-    const endSection = sections[section].end_location;
+    const startSection = sections.tolls[section].start_location;
+    const endSection = sections.tolls[section].end_location;
+
```

Comment:
the find section does not return an array of places, the find section returns an object, and into the object return the tolls path